src/server/models/users.model.js

```diff
@@ -59,7 +59,11 @@ let userSchema = new Schema({
   timezone: Number,
   profile_picture: String,
   job_position: String,
-  date: {
+  reg_date: {
+    type: Date,
+    default: Date.now,
+  },
+  stamp_login: {
     type: Date,
     default: Date.now,
```

Comment:
if stamp_login's default is Date.now, stamp will fixed same with reg_date. Delete this line, and stamp login time manually when login